src/Foundatio/Storage/IFileStorage.cs

```diff
@@ -84,5 +85,43 @@ public static class FileStorageExtensions {
         public static Task<bool> SaveFileAsync(this IFileStorage storage, string path, string contents) {
             return storage.SaveFileAsync(path, new MemoryStream(Encoding.UTF8.GetBytes(contents ?? String.Empty)));
         }
+
+        internal static bool ContainsWildcard(this string searchPattern) {
```

Comment:
I could have sworn we had a string extension for this in foundatio, but maybe that's just in the exceptionless project.